Task_Position_the_Cubes/index.html

```diff
@@ -1,15 +1,23 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Cubes Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-
-    <!-- Start coding from the lines below for POSITION THE CUBES TASK; DO NOT TOUCH the code above-->
-  </body>
-</html>
+<!DOCTYPE html>

```

Comment:
The class name in the opening div tag should be changed from 'buttom' to 'bottom'